docs/data-sources/dashboard_list.md

```diff
@@ -1,23 +1,30 @@
 ---
-page_title: "datadog_dashboard_list"
+page_title: "datadog_dashboard_list Data Source - terraform-provider-datadog"
+subcategory: ""
+description: |-
+  Use this data source to retrieve information about an existing dashboard list, for use in other resources. In particular, it can be used in a dashboard to register it in the list.
 ---
 
-# datadog_dashboard_list Data Source
+# Data Source `datadog_dashboard_list`
 
 Use this data source to retrieve information about an existing dashboard list, for use in other resources. In particular, it can be used in a dashboard to register it in the list.
 
 ## Example Usage
 
-```
+```terraform
 data "datadog_dashboard_list" "test" {
   name = "My super list"
 }
 ```
 
-## Argument Reference
+## Schema
+
+### Required
+
+- **name** (String) A dashboard list name to limit the search.
+
+### Optional
 
--   `name`: (Required) A dashboard list name to limit the search.
+- **id** (String) The ID of this resource.
```

Comment:
I don't see anything using an input `id` field in https://github.com/DataDog/terraform-provider-datadog/blob/master/datadog/data_source_datadog_dashboard_list.go ; how does this ID option work?

Reply:
`id` is the field where terraform stores the resource ID in its state, that you can also reference in other parts of your terraform config.
It's always stored as a string by terraform itself